test/unit/edgeware/testUtil.ts

```diff
@@ -121,21 +121,52 @@ export function constructFakeApi(
     query: {
       system: {
         blockHash: {
-          multi: callOverrides['blockHash.multi'],
+        multi: callOverrides['blockHash.multi'],
```

Comment:
This issue with declaring functions like this is that you will no longer have a "default" call if you try `api.query.system.blockHash()` -- so if any of the .multi or .at or .keysAt calls require preserving the "default" call as well, you will need to declare them using functions as demonstrated at line 94-95.

Reply:
Updated API call to preserve default function calls.